src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -315,23 +315,36 @@ public void testBanditColdStartAction() {
     assertEquals("cold-start-bandit-experiment-bandit", capturedBanditLog.experiment);
     assertEquals("cold-start-bandit", capturedBanditLog.variation);
     assertEquals("subject2", capturedBanditLog.subject);
-    assertEquals(Map.of(), capturedBanditLog.subjectAttributes);
+    assertEquals(Map.of(), capturedBanditLog.subjectNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.subjectCategoricalAttributes);
     assertEquals("option3", capturedBanditLog.action);
-    assertEquals(Map.of(), capturedBanditLog.actionAttributes);
+    assertEquals(Map.of(), capturedBanditLog.actionNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.actionCategoricalAttributes);
     assertEquals(0.3333, capturedBanditLog.actionProbability, 0.0002);
     assertEquals("cold start", capturedBanditLog.modelVersion);
   }
 
   @Test
   public void testBanditModelActionLogging() {
+    // Note: some of the passed in attributes are not used for scoring, but we do still want to make sure they are logged
+
     EppoAttributes subjectAttributes = new EppoAttributes(Map.of(
-      "gender_identity", EppoValue.valueOf("female"),
-      "days_since_signup", EppoValue.valueOf(130) // note: unused for scoring (which looks for account_age)
+    "gender_identity", EppoValue.valueOf("female"),
+    "days_since_signup", EppoValue.valueOf(130), // unused for scoring (which looks for account_age)
+    "is_premium", EppoValue.valueOf(false), // unused for scoring
```

Comment:
Once we settle the [question](https://github.com/Eppo-exp/java-server-sdk-bandit-beta/pull/4#discussion_r1488657718) of categorical value that could cast to numeric (but are not), we should add a test to crystalize that behavior. Could be `"account_tier", EppoValue.valueOf("2")`, for example.